### DISCUSSION TITLE
How to put a single user signature on all pages

### DISCUSSION THREAD

**Discussion by Sanyam1197** [MatthiasValvekens/pyHanko#303] [Q&A]
Hi,

I'm trying to put a signature associated with a stamp on every page of a PDF using pyHanko. I've almost done everything else that I wanted using pyHanko but now I'm stuck on this feature for so long now but I'm still lost can you please help on this matter?

So, basically I want to sign the document only once but that one signature should appear on every page with stamp. I read somewhere that it can't be done directly. So can you please help me with some workaround if possible.

**Comment by MatthiasValvekens** [comment]
See my answer to #305 and (more thoroughly) #70.
